app/services/authService.js

```diff
@@ -75,7 +77,7 @@ class AuthService {
 
   async deleteToken(loginInfoObj) {
     await loginInfoObj.findOneAndDelete();
-    return loginInfoObj; 
+    return loginInfoObj;
```

Comment:
Sometimes the Obj suffix is utilized and sometimes not, why?

Reply:
Here I just followed the old version of the code. So if it was used in old version, I used it too.